File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/11.cpp
#include <iostream>
using namespace std;

/*11. Elabore um algoritmo que solicite ao usuário um número real e ao final imprima na tela o
numero informado e na linha de baixo o dobro deste número da seguinte forma:
Numero -> X
Dobro deste numero -> Y*/

int main(){
	float numero;
	
	printf("Digite um numero real: ");
	scanf("%f", &numero);
	
	printf("\nNumero -> %0.2f", numero);
	printf("\nDobro deste numero -> %0.2f", numero * 2);
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/Python/11.py
"""
11.	Elabore um algoritmo que solicite ao usuário um número real e ao final imprima na tela o 
numero informado e na linha de baixo o dobro deste número da seguinte forma:  
    Numero -> X  
    Dobro deste numero -> Y 
 
"""

floatInput = float(input("Digite um numero real: "))
print("""
Numero -> {:.0f}
Dobro deste numero -> {:.0f}
""".format(floatInput, floatInput * 2))